.github/scripts/install_cuda_centos.sh

```diff
@@ -91,9 +92,12 @@ do :
     if [[ ${package} == libcu* ]] && version_lt "$CUDA_VERSION_MAJOR_MINOR" "11.0" ; then
         package="${package/libcu/cuda-cu}"
     fi
-    # CUDA < 11, -devel- packages were actually -dev
-    if [[ ${package} == *devel* ]] && version_lt "$CUDA_VERSION_MAJOR_MINOR" "11.0" ; then
-        package="${package//devel/dev}"
+    # libnvjitlink not required prior to CUDA 12.0
+    if [[ ${package} == libnvjitlink-dev* ]] && version_lt "$CUDA_VERSION_MAJOR_MINOR" "12.0" ;then
```

Comment:
Not sure if this is as narrow as it could be, but I wanted to make minimal changes to get it working.

The wildcard on `libnvjitlink-dev` can probably be removed.